src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -366,10 +366,10 @@ const ResultsPage = ({ location, tracking }) => {
         <ul>
           <li>Zip Code</li>
         </ul>
-        <p>
+        <div>
```

Comment:
why are you switching this to a div?

Reply:
Console warning for `form` tag being a descendant of the `p` tag

Reply:
ah, because of the other change...gotcha